hack/generator/pkg/codegen/pipeline/status_from_swagger.go

```diff
@@ -57,7 +57,7 @@ func AddStatusFromSwagger(idFactory astmodel.IdentifierFactory, config *config.C
 				return nil, errors.Wrapf(err, "unable to load Swagger data")
 			}
 
-			klog.V(1).Infof("Loaded Swagger data (%d resources, %d other types)", len(swaggerTypes.resources), len(swaggerTypes.otherTypes))
+			klog.V(1).Infof("Loaded Swagger data (%d resources, %d other types)", len(swaggerTypes.ResourceTypes), len(swaggerTypes.OtherTypes))
```

Comment:
Please hold off reviewing this file until I’ve refactored the large functions here.